src/OpenTelemetry/Metrics/Exemplar/ExemplarMeasurement.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#if EXPOSE_EXPERIMENTAL_FEATURES && NET8_0_OR_GREATER
+using System.Diagnostics.CodeAnalysis;
+using OpenTelemetry.Internal;
+#endif
+
+namespace OpenTelemetry.Metrics;
+
+#if EXPOSE_EXPERIMENTAL_FEATURES
+/// <summary>
+/// Represents an Exemplar measurement.
+/// </summary>
+/// <remarks><inheritdoc cref="Exemplar" path="/remarks"/></remarks>
+/// <typeparam name="T">Measurement type.</typeparam>
+#if NET8_0_OR_GREATER
+[Experimental(DiagnosticDefinitions.ExemplarExperimentalApi, UrlFormat = DiagnosticDefinitions.ExperimentalApiUrlFormat)]
+#endif
+public
+#else
+internal
+#endif
+    readonly ref struct ExemplarMeasurement<T>
+    where T : struct
+{
+    internal ExemplarMeasurement(
+        T value,
+        ReadOnlySpan<KeyValuePair<string, object?>> tags)
+    {
+        this.Value = value;
+        this.Tags = tags;
+        this.ExplicitBucketHistogramBucketIndex = -1;
+    }
+
+    internal ExemplarMeasurement(
+        T value,
+        ReadOnlySpan<KeyValuePair<string, object?>> tags,
+        int explicitBucketHistogramIndex)
+    {
+        this.Value = value;
+        this.Tags = tags;
+        this.ExplicitBucketHistogramBucketIndex = explicitBucketHistogramIndex;
+    }
+
+    /// <summary>
+    /// Gets the measurement value.
+    /// </summary>
+    public T Value { get; }
+
+    /// <summary>
+    /// Gets the measurement tags.
```

Comment:
make the comment indicate that this is the full set of tags provided with measurement time

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/5364/commits/8bde052e02ced2f91298fb9b03cd54d7f4ad4e2d